src/pages/auth/login/login.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { Link,useNavigate,useLocation} from "react-router-dom";
+import { AuthContainer } from "../components/authContainer";
+import { signin } from "../../../assets/images";
+import {
+  BsFillEyeFill,
+  FaEnvelope,
+  AiOutlineArrowRight,
+} from "../../../assets/icons";
+import { useAuth } from "../../../context/auth/authContext";
+
+const Login = () => {
+  const [email, setEmail] = useState();
+  const navigate = useNavigate();
+  const { login } = useAuth();
+  const [password, setPassword] = useState();
+  return (
+    <AuthContainer title="Login" imageUrl={signin}>
+      <form className="signup-form-container flex-col">
+        <ul>
+          <li className="list-style-none text-xs">
+            <div className="email-field-container">
+              <label className="text-xs" htmlFor="email">
+                Email address
+              </label>
+
+              <input
+                id="email"
+                className="input-field form-control text-xs"
+                type="email"
+                placeholder="priya@gmail.com"
+                value={email}
+                onChange={(e) => {
+                  setEmail(e.target.value);
+                }}
+                required
```

Comment:
Required tags only work in a form, correct me if i am wrong here

Reply:
I am using form . This is a login form .